### PR TITLE
Coco 118 position fix

### PR DESCRIPTION
**Pull request by dstack**
@aarmour @aaronkaka 

Fix for positioning of coach-marks.

### CODE REVIEW

File: src/js/CoachMark.js [hunk 1]
```diff
@@ -25,8 +25,17 @@ export default class CoachMark {
 			throw new Error('missing required option: ' +
 			'you must specify text for the coach mark');
 
+    // create relative parent for simplified positioning
+    const positioner = document.createElement('div');
+    positioner.style.position = 'relative';
+    positioner.style.display = 'inline-block';
+    //positioner.style.width = '0px';
+    //positioner.style.height = '0px';
```
Comment (aarmour) [inline src/js/CoachMark.js]: Remove commented code.

File: src/js/CoachMark.js [hunk 2]
```diff
@@ -25,8 +25,17 @@ export default class CoachMark {
...
+
+
+
 		//Add style to parent
-		element.style.position = 'relative';
+		//element.style.position = 'relative';
```
Comment (aarmour) [inline src/js/CoachMark.js]: Remove commented code.

**Comment by aarmour** [comment]
There are mixed tabs and spaces.

**Comment by aarmour** [comment]
I haven't dug into this code, but in general tooltip-type components tend to have an option to append the overlay element to `body` instead of the referenced element. If that option were available, would it mitigate positioning issues?

**Comment by dstack** [comment]
This would actually compound the positioning issue, because they we have to re-make the computation in the event of body appending.  I agree the feature should be added, but I think we should add it separately, simply because we have people trying to use it as is, and the primary use case is to be positioned next to a target element.

**Comment by aarmour** [comment]
Moved this to #4 for clean-up and merge. Closing.

**Pull request closed without merging**